src/cli/cli_br.cpp

```diff
@@ -435,6 +435,45 @@ template <> otError Br::Process<Cmd("prefixtable")>(Arg aArgs[])
     return error;
 }
 
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_DHCP6_PD_ENABLE
+template <> otError Br::Process<Cmd("pd")>(Arg aArgs[])
+{
+    otError error = OT_ERROR_NONE;
+
+    VerifyOrExit(aArgs[0].IsEmpty(), error = OT_ERROR_INVALID_ARGS);
+
+    /**
+     * @cli br pd state
+     * @code
+     * br pd state
+     * running
+     * Done
+     * @endcode
+     * @par api_copy
+     * #otBorderRoutingDhcp6PdGetState
+     */
+    if (aArgs[0] == "state")
+    {
+        static const char *const kDhcpv6PdState[] = {"disabled", "stopped", "running"};
```

Comment:
To follow similar style (similar to `Br::Process<Cmd("state")>(Arg aArgs[])`):
```suggestion
        static const char *const kDhcpv6PdStateStrings[] =
        {
            "disabled", // (0) OT_BORDER_ROUTING_DHCP6_PD_STATE_DISABLED
            "stopped",  // (1) OT_BORDER_ROUTING_DHCP6_PD_STATE_STOPPED
            "running",  // (2) OT_BORDER_ROUTING_DHCP6_PD_STATE_RUNNING
        };

```